src/server/pfs/server/val_server.go

```diff
@@ -147,6 +147,13 @@ func (a *validatedAPIServer) SquashCommitSet(ctx context.Context, request *pfs.S
 	return a.apiServer.SquashCommitSet(ctx, request)
 }
 
+func (a *validatedAPIServer) GetFile(request *pfs.GetFileRequest, server pfs.API_GetFileServer) error {
```

Comment:
This should probably also check that `File.Commit`, `File.Commit.Branch`, and `File.Commit.Branch.Repo` are not `nil`.